app/src/main/java/woowacourse/payments/ui/cards/components/PaymentCardsContent.kt

```diff
@@ -69,6 +81,7 @@ private fun SingleCard(
         PaymentCard(
             paymentCard = paymentCard,
             onSelectBank = {},
+            onEditCard = onEditCard,
```

Comment:
```suggestion
            onEditCard = {onEditCard(paymentCard.id)} ,
```
nit: MultiCards와 일관성을 위해 이렇게 하면 어떨까유

Reply:
오옹 좋은 생각이네요~!
수정했습니다!